### PR TITLE
Update material_color_utilities package version to latest 0.11.1

### PR DESCRIPTION
**Pull request by QuncCccccc**
This PR is to update material_color_utilities package version to the latest. `material_color_utilities/scheme/scheme_fruit_salad.dart` and `material_color_utilities/scheme/scheme_rainbow.dart` are exported after version 0.9.0.

Once this PR is merged, we don't need to explicitly import these two files like the change in PR #144805, which breaks some dependencies in `Google testing`.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [x] All existing and new tests are passing.

### CODE REVIEW

File: dev/integration_tests/gradle_deprecated_settings/pubspec.yaml [hunk 1]
```diff
@@ -9,15 +9,15 @@ dependencies:
     sdk: flutter
   camera: 0.10.5+9
 
-  camera_android: 0.10.8+17 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  camera_android: 0.10.8+18 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```
Comment (christopherfujino) [inline dev/integration_tests/gradle_deprecated_settings/pubspec.yaml:12]: This is breaking tool integration tests

Comment (QuncCccccc) [reply dev/integration_tests/gradle_deprecated_settings/pubspec.yaml:12]: Somehow this is still updated after I pinned this package in `update_packages_pins.dart`🧐

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by christopherfujino** [comment]
integration test failure looked like a timeout, scheduled a retry

**Comment by QuncCccccc** [comment]
> integration test failure looked like a timeout, scheduled a retry

Since I still see the timeout error, I rebased master to trigger another rounds of tests, but forget to pull your commit change first. So sorry about that! I just pushed a commit to add the change back to remove the "AUTOGENERATED - ..." line for `camera_android`!

**Comment by christopherfujino** [comment]
Analysis is now failing with:

```
Warning: pubspec in /b/s/w/ir/x/w/flutter/dev/integration_tests/gradle_deprecated_settings has updated or new dependencies. Please run "flutter update-packages --force-upgrade" to update them correctly.
Warning: one or more pubspecs have invalid dependencies. Please run "flutter update-packages --force-upgrade" to update them correctly.
```

You'll have to re-run `flutter update-packages --force-upgrade` to update the checksum at the bottom.

**Comment by QuncCccccc** [comment]
I see. Re-running!

**Comment by QuncCccccc** [comment]
Seems `Windows tool_integration_tests_2_6` still failed because of timeout🧐

**Comment by christopherfujino** [comment]
re-ran windows tool_integration_test_2_6, i highly doubt that this change caused that timeout.

**Comment by christopherfujino** [comment]
> re-ran windows tool_integration_test_2_6, i highly doubt that this change caused that timeout.

I was wrong, revving the frontend_server_client DID indeed regress this test: https://github.com/flutter/flutter/issues/146164

**Comment by crubilar79** [comment]
Hi, I have dependency error:
Dependency problem with flutter_localizations package:
material_color_utilities is pinned to version 0.11.1 by flutter_localizations from the flutter SDK
when I upgrade to version 0.11.1 of the material_color_utilities package it gives other errors:
\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java:34: error: cannot find symbol
      flutterEngine.getPlugins().add(new com.mastertipsy.flutter_localization.FlutterLocalizationPlugin());

symbol:   class FlutterLocalizationPlugin
  location: package com.mastertipsy.flutter_localization
1 error

FAILURE: Build failed with an exception.

**Pull request merged**